packages/react-sdk/README.md

```diff
@@ -28,6 +28,7 @@ These ready-made components can help you quickly build a chat app.
 
 - Node 16.10+
 - React 16.14+
+- For use within this repo, Yarn v3+. See [Yarn Installation](https://yarnpkg.com/getting-started/install).
```

Comment:
Ry, do these clarifications above and below - about yarn being a requirement for use within this repo only - make sense? Did I interpret this correctly? TY!